fastlane/Fastfile

```diff
@@ -940,6 +940,59 @@ REPOSITORY_NAME="WordPress-Android"
     )
   end
 
+  #####################################################################################
+  # create_gh_release
+  # -----------------------------------------------------------------------------------
+  # This lane creates a GitHub release for the current version
+  #   - Attaching the existing .aab files in `build/` as the release assets
+  #   - Using the WP & JP release_notes.txt as description
+  #   - If prerelease:<true|false> is not provided, the pre-release status will be guessed from the version name
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane create_gh_release [app:<wordpress|jetpack>] [version:<Hash{name,code}>] [prerelease:<true|false>]
+  #
+  # Examples:
+  # bundle exec fastlane create_gh_release     # Guesses prerelease status based on version name. Includes existing assets for WPAlpha+WPBeta+JPBeta
+  # bundle exec fastlane create_gh_release app:wordpress prerelease:true                        # Includes existing assets for WPAlpha+WPBeta
+  # bundle exec fastlane create_gh_release version:{name:12.3-rc-4} prerelease:true             # Includes existing assets for WPAlpha+WPBeta+JPBeta 12.3-rc-4
+  # bundle exec fastlane create_gh_release app:jetpack version:{name:12.3-rc-4} prerelease:true # Includes only existing asset for JPBeta 12.3-rc-4
+  #####################################################################################
+  lane :create_gh_release do |options|
+    apps = options[:app].nil? ? ['wordpress', 'jetpack'] : [options[:app]]
+    versions = options[:version].nil? ? [android_get_alpha_version(app: 'wp_jp'), android_get_release_version(app: 'wp_jp')] : [options[:version]]
+
+    release_assets = apps.flat_map do |app|
+      versions.flat_map { |vers| bundle_file_path(app, vers) }
+    end.select { |f| File.exist?(f) }
+
+    release_title = versions.last['name']
+    set_prerelease_flag = options[:prerelease].nil? ? /-rc-|alpha-/.match?(release_title) : options[:prerelease]
+
+    UI.message("Creating release for #{release_title} with the following assets: #{release_assets.inspect}")
+
+    app_titles = { 'wordpress' => 'WordPress', 'jetpack' => 'Jetpack' }
+    tmp_file_file = File.absolute_path('unified-release-notes.txt')
```

Comment:
uhu?  Not sure why I went with that one 🤦 I guess it's because I refactored that part of the code multiple times and ended up not realizing what the end name was 😅  I will definitively update the variable name with your suggestion.

Reply:
Oh I remember now: at first `tmp_file` was a `IO::File`, so when I switched the variable to be just the path, to avoid confusion with past code I wanted to renaming it to `tmp_file_path`. And because I was tired, I wrote `tmp_file_file` instead of `tmp_file_path`… 🤦 

Anyway, fixed in f2a3e704a6eb47da9d58ff527cca6b89e01c0cb4